portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,30 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Anooj!</h1>");
+    ArrayList<String> messages = new ArrayList<String>();
+    messages.add("Hi, my name is Anooj");
+    messages.add("Hello");
+    String json = convertToJson(messages);
+    response.setContentType("application/json");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(ArrayList <String> list) {
```

Comment:
This is totally fine; having said that, I'd probably not bother creating a separate function for this...inlining the behavior is just as readable, I think. But completely up to you.